src/stratigraphy/groundwater/groundwater_extraction.py

```diff
@@ -267,7 +269,7 @@ def get_groundwater_info_from_lines(self, lines: list[TextLine], page: int) -> G
         else:
             raise ValueError("Could not extract all required information from the lines provided.")
 
-    def extract_groundwater(self) -> list[GroundwaterInformationOnPage]:
+    def extract_groundwater(self, elevation: ElevationInformation | None) -> list[GroundwaterInformationOnPage]:
```

Comment:
Maybe naming this `terrain_elevation` or similar would be better, to distinguish it from the groundwater elevation?

Reply:
That sounds like a good idea!